ratpack-cassandra/src/main/java/smartthings/ratpack/cassandra/CassandraModule.java

```diff
@@ -94,14 +93,6 @@ public void setMigrationFile(String migrationFile) {
 			this.migrationFile = migrationFile;
 		}
 
-		public Boolean getAutoMigrate() {
```

Comment:
Why remove auto migration?  Seems like a nice feature to have the ability to disable it?